packaging/hudi-hadoop-mr-bundle/pom.xml

```diff
@@ -29,6 +29,8 @@
   <properties>
     <checkstyle.skip>true</checkstyle.skip>
     <main.basedir>${project.parent.basedir}</main.basedir>
+    <!-- override parquet version to be same as Hive 3.1.2 -->
+    <parquet.version>1.10.1</parquet.version>
```

Comment:
Can we instead add like `hive.parquet.version` in the root POM and use that for the parquet dependencies here ? Changing the version in the internal POMs like this becomes hard to track. If we atleast have a property defined around this, it will be easier to remember in future that we are dealing with multiple parquet versions in the project.

Reply:
@rahil-c adding to Udit's point, do we need to build the hudi-hadoop-mr-bundle for Hive 2.x and 3.x differently, based on the dependency version used?

Reply:
@umehrot2 Will make the change

@yihua Nice catch and yes I believe we need too. In the hive 2.x the parquet is 1.8.x https://github.com/apache/hive/blob/rel/release-2.3.9/pom.xml#L183

In the hive 3.x the parquet is 1.10.x https://github.com/apache/hive/blob/rel/release-3.1.2/pom.xml#L191 

Im thinking we can just set `hive.parquet.version` to be the correct parquet version in the spark2 and spark3 profiles.